modules/ingest-geoip/src/main/java/module-info.java

```diff
@@ -19,5 +19,5 @@
     exports org.elasticsearch.ingest.geoip.direct to org.elasticsearch.server;
     exports org.elasticsearch.ingest.geoip.stats to org.elasticsearch.server;
 
-    exports org.elasticsearch.ingest.geoip to com.maxmind.db;
+    exports org.elasticsearch.ingest.geoip to com.maxmind.db, org.elasticsearch.logstashbridge;
```

Comment:
I can understand that this needs to export to `org.elasticsearch.logstashbridge`, but am unsure why it needs to export _to_ `com.maxmind.db`, which I believe to be a dependency _of_ this module (not dependent _on_ this module).

I'm okay paring this off as a separate thing to chase down.